en/configure-a-tidb-cluster.md

```diff
@@ -557,6 +557,18 @@ The YAML file above:
 
 When Kubernetes deletes the TiDB Pod, it also removes the TiDB node from the service endpoints. This is to ensure that the new connection is not established to this TiDB node. However, because this process is asynchronous, you can make the system sleep for a few seconds before you send the `kill` signal, which makes sure that the TiDB node is removed from the endpoints.
 
+## Configure graceful upgrade for TiKV cluster
```

Comment:
Aligned with the Chinese version, this title should be in the 3rd level.  For "## Configure TiDB cluster graceful upgrade", similarly, please update it to the 3rd level as well.